src/components/issues/IssuesReset.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+
+import { ResetIssuesSort } from 'actions/issues.actions';
+
+type connectedProps = {
+  ResetIssuesSort: () => void
+};
+
+type OwnProps = {
+  sorting: any,
+  issuesState: any
+};
+
+class IssuesReset extends Component<connectedProps & OwnProps> {
+  render() {
+    if (!this.props.sorting && !this.props.issuesState) {
+      return null;
+    }
+
+    return (
+      <Reset onClick={() => this.props.ResetIssuesSort()}>
```

Comment:
no need to create an arrow function. just pass the reference. especially if you are not passing a param